pavelib/assets.py

```diff
@@ -64,6 +64,9 @@
     'cms': [path('lms') / 'static' / 'sass' / 'partials', ],
 }
 
+# Collectstatic log directory setting
+COLLECTSTATIC_LOG_DIR_ARG = "collect_log_dir"
```

Comment:
At least in the function definitions. I know we talked about setting up a `**kwargs` that would or wouldn't include the key as a result of a conditional, but I'd constrain the use as much as possible.